vispy/visuals/isoline.py

```diff
@@ -208,11 +201,11 @@ def draw(self, transforms):
                                                        self._levels)
             self._cl = self._levels_to_colors()
             self._recompute = False
-            self._update_color_lev = False
+            self._update_color_lev = True
```

Comment:
I am confused about this logic--`self._levels_to_colors()` will be called twice?